Documentation/api.md

```diff
@@ -92,6 +92,7 @@ Endpoint defines a scrapeable endpoint serving Prometheus metrics.
 | path | HTTP path to scrape for metrics. | string | false |
 | scheme | HTTP scheme to use for scraping. | string | false |
 | interval | Interval at which metrics should be scraped | string | false |
+| timeout | Timeout after which the scrape is ended | string | false |
```

Comment:
Can we call this `scrapeTimeout` to match the attribute name in the prometheus config?